### ISSUE TITLE
RN 50 support?

### ISSUE THREAD

**Issue by codedre** [react-native-community/react-native-svg#495]
Just upgraded to RN v50 and every one of my SVG components are throwing the following error

```
Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

Check the render method of `G`.
```

```
"react": "16.0.0",
"react-native": "0.50.0",
"react-native-svg": "5.3.2",
```

**Comment by codedre** [comment]
I see that #486 hopes to solve this.

**Comment by brunolemos** [comment]
@codedre merged on 5.5.0, you can closed this

**Comment by kaloncheung124** [comment]
I had the same issue, upgraded my react-native-svg npm package and that solved it for me.
